toplevel/vernac.ml

```diff
@@ -160,7 +160,7 @@ let pr_new_syntax po loc chan_beautify ocom =
    and a glimpse of the executed command *)
 
 let pp_cmd_header loc com =
-  let shorten s = try (String.sub s 0 30)^"..." with _ -> s in
+  let shorten s = try (Unicode.utf8_sub s 0 30)^"..." with _ -> s in
   let noblank s =
```

Comment:
The `try with` was there to capture an exception thrown when the string was less than 30 bytes. With `utf8_sub`, it means that (good news) `try with` is no longer needed and that (bad news) "`...`" is displayed even when the line is not truncated.